pkg/ui/react-app/src/thanos/pages/blocks/Blocks.tsx

```diff
@@ -55,33 +72,45 @@ export const BlocksContent: FC<{ data: BlockListProps }> = ({ data }) => {
   };
 
   if (err) return <UncontrolledAlert color="danger">{err.toString()}</UncontrolledAlert>;
-
+  const date = new Date(refreshedAt).toLocaleString().split(',');
   return (
     <>
       {blocks.length > 0 ? (
-        <div className={styles.container}>
-          <div className={styles.grid}>
-            <div className={styles.sources}>
-              {Object.keys(blockPools).map((pk) => (
-                <SourceView
-                  key={pk}
-                  data={blockPools[pk]}
-                  title={pk}
-                  selectBlock={selectBlock}
-                  gridMinTime={viewMinTime}
-                  gridMaxTime={viewMaxTime}
-                />
-              ))}
+        <div>
+          <div className={styles.blockStats}>
+            <span ref={linkRef}>Last Refresh on {date[0]}</span>
+            {ready && (
```

Comment:
We actually don't need `ready`, `setReady` and the corresponding `useEffect`, using `linkRef.current` directly here would suffice.

```suggestion
            {linkRef.current && (
```